partcad/src/partcad/ai_ollama.py

```
@@ -71,18 +72,16 @@ def generate_ollama(
         if not ollama_once():
             return None
 
-        image_content = []
+        def handle_content(content):
```

Ruff (0.8.2)

Missing return type annotation for private function `handle_content`

(ANN202)